testApp/android/app/src/androidTest/java/com/example/reactnativedidomi/UIGettersTest.kt

```diff
@@ -157,4 +157,18 @@ class UIGettersTest: BaseUITest() {
         assertTextContains(",\"global_consent\":{\"enabled\":[".trim())
         assertTextContains(",\"consent\":{\"enabled\":[".trim())
     }
+
+    @Test
+    fun test_getUserStatus_vendors_globalConsent() {
```

Comment:
For consistency, method name could be test_getUserStatus_vendors_global_consent

Reply:
change pushed